core-java-modules/core-java-collections-list-4/src/test/java/com/baeldung/list/creation/ArraysAsListUnitTest.java

```diff
@@ -0,0 +1,57 @@
+package com.baeldung.list.creation;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ArraysAsListUnitTest {
+
+    @Test
+    void givenAnArrayOfIntegersWhenCreatingAListUsingArraysAsListThenItWillHaveTheSameElementsInTheSameOrder() {
+        Integer[] array = new Integer[]{1, 2, 3, 4};
+        List<Integer> list = Arrays.asList(array);
+        assertThat(list).containsExactly(1, 2, 3, 4);
+    }
+
+    @Test
+    void givenAnListOfStringCreatedUsingArraysAsListWhenChangingTheOriginalArrayTheReturnListWillAlsoChange() {
+        Integer[] array = new Integer[]{1, 2, 3};
+        List<Integer> list = Arrays.asList(array);
+        array[0] = 1000;
+        assertEquals(1000, list.get(0));
```

Comment:
Personally, I'd prefer to see `assertThat` used throughout these tests, but this is valid